integration-tests/src/test/java/org/openmrs/module/fhir2/providers/r4/EncounterFhirResourceProviderIntegrationTest.java

```diff
@@ -90,10 +89,6 @@ public class EncounterFhirResourceProviderIntegrationTest extends BaseFhirR4Inte
 	@Override
 	public void setup() throws Exception {
 		super.setup();
-		
-		for (String encounterData : ENCOUNTER_DATA_XML) {
-			executeDataSet(encounterData);
-		}
```

Comment:
Note that by removing this, most of these test by default run against *just* the standard test data, and i now only load additional test data as needed.  See my comment below.